manifests/server.pp

```diff
@@ -39,22 +43,79 @@
     }
   }
 
+
   if $use_xinetd {
-    include xinetd
-    xinetd::service { 'rsync':
-      bind        => $address,
-      port        => '873',
-      server      => '/usr/bin/rsync',
-      server_args => "--daemon --config ${conf_file}",
-      require     => Package['rsync'],
+    include ::xinetd
+    if $enable_ipv4 {
```

Comment:
This does not follow the style guide https://puppet.com/docs/puppet/5.5/style_guide.html#simple-resource-declarations